web/pgadmin/dashboard/templates/dashboard/sql/default/pgd_cluster_nodes.sql

```diff
@@ -0,0 +1,12 @@
+with node_details as (
+    SELECT node_id, version() pg_version, bdr.bdr_version() bdr_version
+    FROM bdr.local_node
+)
+select ns.*, nd.*,
+    ni.catchup_state_name
+from bdr.node_summary ns
+    left join node_details nd
+    on ns.node_id = nd.node_id
+    left join bdr.node_catchup_info_details ni
+    on ns.node_id = ni.target_node_id
+    order by node_seq_id;
```

Comment:
Keywords (select, from, left join, on, order by) should be in the capital for consistency.